scipy/integrate/tests/test_bvp.py

```diff
@@ -532,6 +602,41 @@ def test_big_problem():
     assert_allclose(sol.sol(sol.x, 1), sol.yp, rtol=1e-10, atol=1e-10)
 
 
+def test_big_problem_with_parameters():
+    n = 30
+    x = np.linspace(0, np.pi, 5)
+    x_test = np.linspace(0, np.pi, 100)
+    y = np.ones((2 * n, x.size))
+
+    for fun_jac in [None, big_fun_with_parameters_jac]:
+        for bc_jac in [None, big_bc_with_parameters_jac]:
+            sol = solve_bvp(big_fun_with_parameters, big_bc_with_parameters, x, y, p=[0.5, 0.5],
+                            fun_jac=fun_jac, bc_jac=bc_jac)
+
+            assert_equal(sol.status, 0)
+            assert_(sol.success)
+
+            assert_allclose(sol.p, [1, 1], rtol=1e-4)
+
+            sol_test = sol.sol(x_test)
+
+            for isol in range(0, n, 4):
+                assert_allclose(sol_test[isol], big_sol_with_parameters(x_test, [1, 1])[0],
```

Comment:
Also seem to violate 80 lines. Consider splitting into 3 lines.

(Check other places ideally).

Reply:
Done. I located all lines >80 and changed them, there were 4 lines in all. (There were 2 other lines >80 that pre-existed, i.e. were not my code. I didn't change those because I felt that would be scope creep of this PR. If you thought it worthwhile, I could do a separate PR to bring all lines in scipy.integrate within 80 characters.)